cypress/support/utils/builder.js

```diff
@@ -75,6 +75,41 @@ export class Builder {
     });
   }
 
+  static createBasicProductEntitiesWithBusinessPartnerAndCUTUAllocation(
```

Comment:
Are we gonna reuse this anywhere else that it landed in the builder ?

Reply:
I thought I can use it in another test but actually I don't need it you'e right, I will remove it.